.cloudbuild/cloudbuild.yaml

```diff
@@ -12,48 +12,11 @@ artifacts:
 steps:
```

Comment:
if we can up the timeout at the top of this file to 2.5/3 hours for now till we remove some of the signing, I think that would be helpful to prevent the build from failing unnecessarily.